src/components/SubmitAnswerForm.js

```diff
@@ -23,9 +23,11 @@ const mutation = graphql`
 export default function SubmitAnswerForm ({ close, questionId, choice }) {
   const {
     errors,
-    handleChange,
     handleSubmit,
-    isLoading
+    handleChange,
+    isLoading,
+    selection,
```

Comment:
You can remove this custom `selection` state, and instead use the generic `input` state which contains the entire form.